src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -9,16 +9,69 @@ interface Props {
 }
 
 const OwnerPanel: React.SFC<Props> = ({ imageSrc, ownerName, className }) => {
+  
+  // these will be props fix throughout when passed thru 
+  const current = 4000;
+  const need = 5000;
+  //  fix ^^^ 
+
+  
   return (
     <section className={classnames(styles.container, className)}>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
```

Comment:
What's this?

Reply:
to import social media icons

Reply:
we may want to consider having this live somewhere else so it could be imported into any component

Reply:
moved to the index.html instead! all components can use now